src/Sarif/LineInfo.cs

```diff
@@ -51,7 +51,7 @@ public LineInfo(int startOffset, int lineNumber)
         /// <returns>true if the objects are considered equal, false if they are not.</returns>
         public override bool Equals(object obj)
         {
-            LineInfo? other = obj as LineInfo?;
```

Comment:
>LineInfo [](http://example.com/codeflow?start=12&length=8)

here it is friends, the application of our var standard. we do not require the type to be expressed if the type is already quite explicit. we do this to reduce the # of chars on a line to and allow left-aligning var declarations nicely for this circumstance.